src/hotspot/share/prims/jvm.cpp

```diff
@@ -3866,6 +3867,24 @@ JVM_ENTRY_NO_ENV(jlong, JVM_GetRandomSeedForDumping())
   }
 JVM_END
 
+JVM_ENTRY(jboolean, JVM_IsDumpingClassList(JNIEnv *env))
+  JVMWrapper("JVM_IsDumpingClassList");
+  return DumpLoadedClassList != NULL && classlist_file->is_open();
```

Comment:
For sanity, it's better to add `classlist_file != NULL`

Reply:
done